scripts/flattener.py

```diff
@@ -333,67 +344,65 @@ def get_sex_ontology(donor_df):
 
 # Make sure cxg_adata and cxg_adata_raw have same number of features
 # If not, add implied zeros to csr, and add corresponding 'feature_is_filtered'
-def add_zero():
-	global cxg_adata
-	global cxg_adata_raw
-	if cxg_adata_raw.shape[1] > cxg_adata.shape[1]:
-		genes_add = [x for x in cxg_adata_raw.var.index.to_list() if x not in cxg_adata.var.index.to_list()]
-		new_matrix = sparse.csr_matrix((cxg_adata.X.data, cxg_adata.X.indices, cxg_adata.X.indptr), shape = cxg_adata_raw.shape)
-		all_genes = cxg_adata.var.index.to_list()
+def add_zero(GLOBAL_OBJS):
+	if GLOBAL_OBJS.CXG_ADATA.cxg_adata_raw.shape[1] > GLOBAL_OBJS.CXG_ADATA.cxg_adata.shape[1]:
+		genes_add = [x for x in GLOBAL_OBJS.CXG_ADATA.cxg_adata_raw.var.index.to_list() if x not in GLOBAL_OBJS.CXG_ADATA.cxg_adata.var.index.to_list()]
+		new_matrix = sparse.csr_matrix((GLOBAL_OBJS.CXG_ADATA.cxg_adata.X.data, GLOBAL_OBJS.CXG_ADATA.cxg_adata.X.indices, \
+			GLOBAL_OBJS.CXG_ADATA.cxg_adata.X.indptr), shape = GLOBAL_OBJS.CXG_ADATA.cxg_adata_raw.shape)
```

Comment:
Seeing how this line of code would look if we stored everything under global:
`new_matrix = sparse.csr_matrix((glob.cxg_adata.X.data, glob.cxg_adata.X.indices, glob.cxg_adata.X.indptr), shape = glob.cxg_adata_raw.shape)`